checkstyle/checkstyle.xml

```diff
@@ -60,10 +60,9 @@
     <module name="TypeName"/>
     
     <!-- whitespace -->
-    <module name="GenericWhitespace"/>
```

Comment:
Was able to disable checkStyle for `WindowOperatorImpl.java`. 

For future reference, we can now use `// CHECKSTYLE:OFF` to disable checkstyle checks for specific files.